parquet/src/main/java/org/apache/iceberg/parquet/BaseColumnIterator.java

```diff
@@ -72,4 +73,7 @@ public boolean hasNext() {
     return triplesRead < triplesCount;
   }
 
+  public void setRowPosition(long rowPosition) {
```

Comment:
Instead of adding this, can you update `setPageSource` like the other interface that changed?